schema/deploy/shipping/views.sql

```diff
@@ -728,4 +728,53 @@ comment on view shipping.return_results_v2 is
     'Version 2 of view of barcodes and presence/absence results for return of results on website';
 
 
+create or replace view shipping.hcov19_observation_with_presence_absence_result_v1 as
```

Comment:
> What do you think about making this name a bit simpler? `shipping.hcov19_observation` seems like a nicer alternative.

That sounds great to me! I was trying to be consistent with the `observation_` view naming. 

>     * Because of those things, I would really like to stop using the `shipping.observation_with_presence_absence_result` views for new things moving forward. I don't think it's a well-designed query (it doesn't de-duplicate p/a results, for example), and I realize that's 100% my own doing.

Good to know! I didn't realize you wanted to ditch this view. 

>     * Finally, the columns and data in this view don't seem to me to bear much resemblance to `shipping.observation_with_presence_absence_result`.

Fair enough. 

I'm happy to drop all mention of `observation_with_presence_absence_result`.

Reply:
> Good to know! I didn't realize you wanted to ditch this view.

Yeah, and to be clear, totally didn't expect you to. :-) Just something I've been ruminating on, but that's been super low priority.